### ISSUE TITLE
Dark mode for the rest of the UI

### ISSUE THREAD

**Issue by dflock** [johnfactotum/foliate#470]
Dark mode while reading the book is great, but the rest of the UI is light. An option to switch this to dark mode would be great - or do I have to do this as the system level?

```
➜ neofetch --backend off

duncan@duncan-desktop 
--------------------- 
OS: Ubuntu 18.04.4 LTS x86_64 
Kernel: 4.15.0-106-generic 
Uptime: 1 day, 6 hours, 8 mins 
Packages: 3713 (dpkg), 22 (flatpak), 2 (snap) 
Shell: bash 4.4.20 
Resolution: 1920x1080, 1920x1080 
DE: Xfce 
WM: Xfwm4 
WM Theme: Arc-Dark 
Theme: Arc-Dark [GTK2], Greybird [GTK3] 
Icons: Papirus-Dark [GTK2], elementary-xfce-darker [GTK3] 
Terminal: kitty 
CPU: Intel i7-6700K (8) @ 4.200GHz 
GPU: Intel HD Graphics 530 
Memory: 6902MiB / 64305MiB 
```

**Comment by johnfactotum** [comment]
It does set the rest of the UI to dark, by setting `gtk-application-prefer-dark-theme` to `TRUE`. But it depends on the GTK theme to implement a dark variant.

> OS: Ubuntu 18.04.4 LTS x86_64
> Theme: Arc-Dark [GTK2], Greybird [GTK3] 

Older versions of Greybird doesn't have a dark variant, but I think the version in 18.04 does. So that's probably not the issue.

So I'm guessing you're using the Flatpak version. For some reason it uses Adwaita's light variant even though a dark variant is available. What I think is happening is this: it reads the host settings, and tries to use Greybird; but Greybird is not available in the runtime, so it's actually falling back to GTK's default styling, which is the light variant of Adwaita.

Indeed, this is not specific to Flatpak. For any GTK application, if you run it with `GTK_THEME=no-such-theme`, it will fallback to Adwaita light and `gtk-application-prefer-dark-theme` will have no effect. This is probably a GTK bug.

Anyway, it can be solved by installing the GTK theme (Greybird, in this case) as Flatpak:
```
flatpak install flathub org.gtk.Gtk3theme.Greybird
```

**Comment by dflock** [comment]
> So I'm guessing you're using the Flatpak version.

Correct:

```
 ➜ flatpak info com.github.johnfactotum.Foliate

Foliate - A simple and modern eBook viewer

          ID: com.github.johnfactotum.Foliate
         Ref: app/com.github.johnfactotum.Foliate/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 2.3.0
     License: GPL-3.0-or-later
      Origin: flathub
  Collection: org.flathub.Stable
Installation: system
   Installed: 5.1 MB
     Runtime: org.gnome.Platform/x86_64/3.36
         Sdk: org.gnome.Sdk/x86_64/3.36

      Commit: d56e8456a3848d245010ad912b0de4da18bac9462003fcc290a0da74333df366
      Parent: 6851507c37efbd50844f5fa352b350e73e788edb02bb26327fe2d95780ce536b
     Subject: Update to 2.3.0 (3156996a)
        Date: 2020-06-18 18:18:14 +0000
```

However:

```
 ➜ flatpak install flathub org.gtk.Gtk3theme.Greybird
Looking for matches…
Skipping: org.gtk.Gtk3theme.Greybird/x86_64/3.22 is already installed
```

**Comment by johnfactotum** [comment]
Can you post a screenshot? What Foliate theme are you using?

**Comment by dflock** [comment]
![Screenshot_2020-07-01_21-06-13](https://user-images.githubusercontent.com/47756/86315338-b90d1e80-bbde-11ea-8e8d-934879f98737.png)
![Screenshot_2020-06-30_14-49-07](https://user-images.githubusercontent.com/47756/86315340-b9a5b500-bbde-11ea-91fe-db602152b40d.png)
![Screenshot_2020-06-30_14-48-41](https://user-images.githubusercontent.com/47756/86315341-ba3e4b80-bbde-11ea-86e3-b6e28091868a.png)
![Screenshot_2020-06-30_14-48-25](https://user-images.githubusercontent.com/47756/86315342-ba3e4b80-bbde-11ea-89bb-62fbd9a66e15.png)

**Comment by johnfactotum** [comment]
It's still using the fallback theme, which is always light. It's not using Greybird even though you said it's installed.

**Comment by dflock** [comment]
```
➜ flatpak info org.gtk.Gtk3theme.Greybird

Greybird GTK+ Theme - Greybird GTK+ theme

          ID: org.gtk.Gtk3theme.Greybird
         Ref: runtime/org.gtk.Gtk3theme.Greybird/x86_64/3.22
        Arch: x86_64
      Branch: 3.22
      Origin: flathub
  Collection: org.flathub.Stable
Installation: system
   Installed: 516.1 kB

      Commit: 2cf601519d1fe09f070d3eba242cf0bd5709617d5e5f1b58bf6b176019ead347
      Parent: a5a1a09bf9ae5bab9aec8b805f69273add9c5eef9e7cd908d8d5f713972fbaa0
     Subject: Updated to version 3.22.12 (425f534b)
        Date: 2020-03-23 02:57:59 +0000
```

Is there anything else that I can do to help debug this?

**Comment by johnfactotum** [comment]
Maybe something else on your system is overriding the theme? Why are you using different Arc-Dark with Greybird? Perhaps installing the Flatpak theme for Arc-Dark would help?

Or maybe something's wrong with your Flatpak setup. From the output you posted it looks like you're using a newer version of Flatpak? Which might or might not be related to the issue.

Anyway, I don't really have any ideas. In any case this isn't a Foliate issue.

**Comment by dflock** [comment]
Thanks for your help and responsiveness!

After a bit of fiddling around, I eventually fixed it (as you said) like this:

```
flatpak install flathub org.gtk.Gtk3theme.Arc-Dark
```

**Comment by vnbubba** [comment]
Please add dark mode with Nordic theme. 
https://www.xfce-look.org/p/1267246/

**Comment by vnbubba** [comment]
hi @johnfactotum, I just fixed it. The Foliate I installed is snap version. When I reinstalled by add ppa and install it from repository as
```
sudo add-apt-repository ppa:apandada1/foliate
sudo apt install foliate
```
It's work again! 
https://imgur.com/vD6KsH8.png

**Comment by newmankyle** [comment]
I thought I'd share how I fixed this.
tldr: my GTK_THEME was malformed `sudo flatpak override --env=GTK_THEME=Adwaita:dark` or even `sudo flatpak override --env=GTK_THEME=` fixed this for me.

I tried installing a GtkTheme like listed above, but I assume that was for Foliate 2?

For debugging, `flatpak run com.github.johnfactotum.Foliate` returns this
```
(com.github.johnfactotum.Foliate:2): Gtk-WARNING **: 11:39:37.435: Unknown key gtk-modules in /<my-home>/.var/app/com.github.johnfactotum.Foliate/config/gtk-4.0/settings.ini

(com.github.johnfactotum.Foliate:2): Adwaita-WARNING **: 11:39:37.524: Using GtkSettings:gtk-application-prefer-dark-theme with libadwaita is unsupported. Please use AdwStyleManager:color-scheme instead.
```
googling `gtk-application-prefer-dark` is what led me to adjusting the GTK_THEME
